lighthouse-core/report/report-generator.js

```diff
@@ -151,8 +151,10 @@ class ReportGenerator {
       }
 
       const partials = reportPartials.report.partials;
-      Handlebars.registerPartial(audit.name,
-        Handlebars.template(partials[audit.name] || partials['null-formatter']));
+      const partial = partials[audit.name] ||
+          partials[audit.extendedInfo.formatter] ||
```

Comment:
Am sorry for this. 😞  Wonder how `crc` was working 
@paulirish null will not work here. you have to pass it as a template object.